.github/workflows/sync.yml

```
@@ -146,7 +146,7 @@ jobs:
           RCLONE_CONF: ${{ secrets.RCLONE_CONF }}
           RCLONE_GCS_SERVICE_ACCOUNT_CREDENTIALS: ${{ secrets.RCLONE_GCS_SERVICE_ACCOUNT_CREDENTIALS }}
         with:
-          args: "sync -c -v gs:cmip6 s3:cmip6-pds --filter '- /ScenarioMIP/{CCCma,CNRM-CERFACS,IPSL,EC-Earth-Consortium,MIROC,NCAR,MPI-M,OHC,MRI,DKRZ}/**' --filter '+ /ScenarioMIP/**' --filter '- *' --transfers 15 --checkers 65"
+          args: "sync -c -v gs:cmip6 s3:cmip6-pds --filter '- /CMIP6/ScenarioMIP/{CCCma,CNRM-CERFACS,IPSL,EC-Earth-Consortium,MIROC,NCAR,MPI-M,OHC,MRI,DKRZ}/**' --filter '+ /ScenarioMIP/**' --filter '- *' --transfers 15 --checkers 65"
```

```suggestion
          args: "sync -c -v gs:cmip6 s3:cmip6-pds --filter '- /CMIP6/ScenarioMIP/{CCCma,CNRM-CERFACS,IPSL,EC-Earth-Consortium,MIROC,NCAR,MPI-M,OHC,MRI,DKRZ}/**' --filter '+ /CMIP6/ScenarioMIP/**' --filter '- *' --transfers 15 --checkers 65"
```

Slight tweak to filter

---

yup